src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -54,7 +54,12 @@ public static FormulaValue FromJson(JsonElement element, FormulaType formulaType
                     return FormulaValue.NewBlank(formulaType);
 
                 case JsonValueKind.Number:
-                    if ((skipTypeValidation && numberIsFloat) || formulaType is NumberType)
+                    if (formulaType is DateTimeType)
+                    {
+                        // unixtime

```

Comment:
>unixtime [](http://example.com/codeflow?start=27&length=8)

There are lots of ways to convert between date/number - is this specified in the swagger spec?
Or do we need some additional metadata in the swagger to know which conversion format to use?